### ISSUE TITLE
[Feature suggest] Mkv chapter Import&Output support

### ISSUE THREAD

**Issue by cyzs233** [mifi/lossless-cut#1131]
Basically, this will make lossless an interactive chapter editor. 
No more cumbersome copy&paste timecode from video player into mkvtoolnix's chapter editor, now you can actually preview your chapters live in lossless_cut and align them perfectly to keyframes!

Some features in my mind:
- Import chapter .txt file, and convert each chapter entry into segments. (Ideally use chapter name as segment label)
- Output segments as chapter .txt file. Preferably use "Chapter $$" as the default template for segments without labels.
- Load input video file's chapter directly.

**Comment by mifi** [comment]
losslesscut already supports embedded mp4 chapters. it will ask about importing chapters when opening a file. I think also mkv embedded chapters should be supported. does it not work? do you have an example file with such chapters?

**Comment by cyzs233** [comment]
> losslesscut already supports embedded mp4 chapters. it will ask about importing chapters when opening a file. I think also mkv embedded chapters should be supported. does it not work? do you have an example file with such chapters?

Yeah, chapter is detected as shown in the tracks section. But I thought it would be a continent to be able to convert chapters into LLC format so that you can clip or edit an existing chapter easily.

**Comment by mifi** [comment]
I just tested an mkv file with embedded chapters and they are successfully imported as segments in losslesscut. If you have a small file that does not work, then maybe you could share it and i can see why it doesn't work

**Comment by cyzs233** [comment]
> I just tested an mkv file with embedded chapters and they are successfully imported as segments in losslesscut. If you have a small file that does not work, then maybe you could share it and i can see why it doesn't work

Ops, I realized that I forget to tick the "ask for chapter" option. The import part working fine right now.👍
Any chance "export as chapter" would be supported. Or if it's possible to update embedded chapters on the fly? 
Most chapters that I'm working on are chapters extracted from old DVD, and they need minor adjustments to match newer Bluray/WebDL in face of different cuts/runtime.

**Comment by mifi** [comment]
great! you can use the special chapters only export mode for that

<img width="131" alt="Screenshot 2022-05-09 at 12 46 03" src="https://user-images.githubusercontent.com/402547/167486356-3452dbfb-5f68-41e7-8d53-eaa9c2a448da.png">

**Comment by cyzs233** [comment]
Thanks!! That looks like is the thing I'm requesting.
Why not move it to the "Export project" section though🤣

**Comment by mifi** [comment]
yea the naming is a bit confusing. i guess the export video button also could be moved to export project. have to go through all the naming in the future
